packages/react-vapor/src/components/tooltip/Tooltip.tsx

```diff
@@ -103,6 +104,7 @@ export class Tooltip extends React.Component<ITooltipProps> {
             <BootstrapTooltip
                 id={_.uniqueId('tooltip-')}
                 ref={this.overlay}
+                container={this.props.container || 'body'}

```

Comment:
add the value in DEFAULTS variable, if someone want to redefined the default container for the lib, They can change te static value to impact all tooltip ( Like the Drop root )